### PR TITLE
added/reworked content to robotics and xlab projects

### PR DESCRIPTION
**Pull request by fraujaeger**

### CODE REVIEW

**Comment by alexabruck** [comment]
@fraujaeger How come there are 83 files changed? :-)

**Comment by fraujaeger** [comment]
I don't know why there are seemingly changes in every file. The files i didn't touch are exactly the same of the main branch.

**Comment by fraujaeger** [comment]
Should i do another pull request without the files i didn't change?

**Comment by alexabruck** [comment]
@fraujaeger So you didn't change the files? I wonder why git detects a change in them.. Could you find out? Maybe its the result of some auto-formatting you activated in your code editor (e.g. adding spaces or tabs etc.)

**Comment by simonmaris** [comment]
this was due to a stupid problem with CRLF and LF line endings being different between macos and windows.. 

After working through the git documentation / stackoverflow, I was slightly confused, what would be the way to go. This post helped clear things up a little bit: https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/

I have now normalized everything and introduced a .gitattributes file, which should take care of this with and for all current and future contributors.. 

So, I guess, problem fixed.. Time to review this giant PR :)

File: Collaborative Robot.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+aliases: cobot
+tags: industrial robot UR
```
Comment (simonmaris) [inline Collaborative Robot.md:3]: these tags are all interpreted, no comma necessary. However, this means we currently have the tags _robot_, _robotics_ and _Robotic_, not very clear.. you can enable the core plugin _Tag Pane_ in Obsidian to get an overview and sort this a little bit..

File: Control Box.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	-  Linux operating system
+	-  URs programming interface Polyscope
+	-  all of the users programms
+
+![Control Box](/assets/media/control_box.jpg)
+
+<div style="text-align:right"> UR acadamy </div>
+
+
+*Learn more about the Control Box at [UR acadamy](https://academy.universal-robots.com/free-e-learning/cb3-e-learning/) Lesson 1.*
```
Comment (simonmaris) [inline Control Box.md:20]: typo

**Review by simonmaris — CHANGES_REQUESTED** [review]
I skimmed through most files, content looks good, the general points as commented are:

- no html, let's find another way for image captions
- clearer use of tags (and aliases)

**Pull request merged**